app/scripts/lib/account-tracker.ts

```diff
@@ -265,16 +353,15 @@ export default class AccountTracker {
    * Returns the accounts object for the chain ID, or initializes it from the globally selected
    * if it doesn't already exist.
    *
-   * @private
-   * @param {string} chainId - The chain ID
+   * @param chainId - The chain ID
    */
-  #getAccountsForChainId(chainId) {
+  #getAccountsForChainId(chainId: Hex) {
     const { accounts, accountsByChainId } = this.store.getState();
     if (accountsByChainId[chainId]) {
       return cloneDeep(accountsByChainId[chainId]);
     }
 
-    const newAccounts = {};
+    const newAccounts: Record<string, Record<string, never>> = {};
```

Comment:
can we use `AccountTrackerState['accounts'] ` like we used [here](https://github.com/MetaMask/metamask-extension/blob/1b07a49d095519789b12f6cf0c56852e2ceebad4/app/scripts/lib/account-tracker.ts#L698) ?